dbt-cta/quickbooks/models/1_cta_full_refresh/_1_cta_full_refresh__models.yml

```diff
@@ -0,0 +1,6918 @@
+version: 2
+
+models:
+
+  - name: tax_agencies_MetaData_base
+    description: ''
+    columns:
+      - name: _airbyte_tax_agencies_hashid
+        description: ''
+      - name: createtime
+        description: ''
+      - name: lastupdatedtime
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+      - name: _airbyte_emitted_at
+        description: ''
+      - name: _airbyte_normalized_at
+        description: ''
+      - name: _airbyte_MetaData_hashid
+        description: ''
+        tests:
+          - unique
+
+  - name: items_IncomeAccountRef_base
+    description: ''
+    columns:
+      - name: _airbyte_items_hashid
+        description: ''
+      - name: name
+        description: ''
+      - name: value
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+      - name: _airbyte_emitted_at
+        description: ''
+      - name: _airbyte_normalized_at
+        description: ''
+      - name: _airbyte_IncomeAccountRef_hashid
+        description: ''
+        tests:
+          - unique
+
+  - name: estimates_TxnTaxDetail_TaxLine_TaxLineDetail_base
+    description: ''
+    columns:
+      - name: _airbyte_TaxLine_hashid
+        description: ''
+      - name: percentbased
+        description: ''
+      - name: taxrateref
+        description: ''
+      - name: netamounttaxable
+        description: ''
+      - name: taxpercent
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+      - name: _airbyte_emitted_at
+        description: ''
+      - name: _airbyte_normalized_at
+        description: ''
+      - name: _airbyte_TaxLineDetail_hashid
+        description: ''
+        tests:
+          - unique
+
+  - name: journal_entries_CurrencyRef_base
+    description: ''
+    columns:
+      - name: _airbyte_journal_entries_hashid
+        description: ''
```

Comment:
Singular so looks to be unique. https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/journalentry